app/controllers/cards_controller.rb

```diff
@@ -1,6 +1,9 @@
 class CardsController < ApplicationController
   before_action :find_card, only: [:show, :edit, :update, :destroy]
 
+  require 'nokogiri'
+  require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.